test/e2e/docker/Dockerfile.debug

```diff
@@ -20,6 +20,7 @@ RUN go mod download
 COPY . .
 RUN echo $COMETBFT_BUILD_OPTION && make build && cp build/cometbft /usr/bin/cometbft
 COPY test/e2e/docker/entrypoint* /usr/bin/
+COPY test/e2e/docker/entrypoint-builtin-delve /usr/bin/entrypoint-builtin
```

Comment:
These two lines mean there will be an exact copy of `entrypoint-builtin-delve` named `entrypoint-builtin`. Is this intended? Maybe rename it in the source instead?

Reply:
reverted this change, which was for debugging.
copying was a way to override the entry point.